skyvern/forge/sdk/workflow/context_manager.py

```
@@ -52,6 +53,8 @@
 
 jinja_sandbox_env = SandboxedEnvironment()
 
+RANDOM_SECRET_ID_PREFIX = "placeholder_"
```

Ruff (0.14.11)

Possible hardcoded password assigned to: "RANDOM_SECRET_ID_PREFIX"

(S105)